user_templates/mcu_defaults/main/BTT_Octopus.cfg

```diff
@@ -44,4 +44,3 @@ aliases:
     LIGHT_OUTPUT=MCU_HE2         ,
     LIGHT_NEOPIXEL=MCU_STOP5     ,
     STATUS_NEOPIXEL=MCU_NEOPIXEL ,
-
```

Comment:
Can you put back this second free new line at the end? It was there on purpose to allow better separation when injecting the templates from the install script